### DISCUSSION TITLE
First payment is stuck

### DISCUSSION THREAD

**Discussion by encse** [community/community#54961] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I got a one-time sponsorship back in december, this is the one and only one so far. I see the money in Stripe Connect as 'on the way'.  But it's May already....

It's not clear to me if 
- some information is missing on my side or
- GitHub is keeping back the money for some reason (is there a payout limit?), 
- the supporter changed his mind
- what else

How can I sort this out?

**Comment by DHayes10** [comment]
Hi @encse, 

Thank you for reaching out. I'm sorry you're having issues receiving your sponsorship. Please allow me some time to research this issue. 

Thank you, 
Deona 
GitHub Sponsors

**Reply by dancan2550-code** [reply]
Hi how can retrieve my funds because the recipient didn't receive it

**Comment by DHayes10** [comment, marked as answer]
> ### Select Topic Area
> Question
> 
> ### Body
> I got a one-time sponsorship back in december, this is the one and only one so far. I see the money in Stripe Connect as 'on the way'. But it's May already....
> 
> It's not clear to me if
> 
> * some information is missing on my side or
> * GitHub is keeping back the money for some reason (is there a payout limit?),
> * the supporter changed his mind
> * what else
> 
> How can I sort this out?

Hi @encse, 

It appears your Stripe account was pending. I was able to verify and update the information. I issued a manual payout for the current balance. This should be reflected in your account within the next few business days. Please don't hesitate to reach out if you have any other concerns. 

Thank you, 
Deona 
GitHub Sponsors

**Comment by encse** [comment]
Sounds great, thanks!

**Comment by Bhavyajain21** [comment]
Hey @DHayes10 , I am facing a similar issue at my end and I am unable to do a manual payout.

Would appreciate your help!
